src/cli/node/json_rpc_client.rs

```diff
@@ -345,6 +347,233 @@ pub fn master_key_export(
+    block_hash_data_request_tx: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    block_hash_tally_tx: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    data_request_state: Option<DataRequestState>,
+    /// [(pkh, reveal_value, "rewarded" | "penalized" | "")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    reveals: Option<Vec<(String, String, &'static str)>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    tally: Option<String>,
+}
+
+#[derive(Debug, Serialize)]
+struct DataRequestState {
+    stage: String,
+    current_commit_round: u16,
+    current_reveal_round: u16,
+}
+
+impl fmt::Display for DataRequestTransactionInfo {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        writeln!(f, "Report for data request {}:", self.data_request_tx_hash)?;
+        writeln!(
+            f,
+            "data_request_output: {}",
+            serde_json::to_string_pretty(&self.data_request_output).unwrap()
+        )?;
+
+        if self.block_hash_data_request_tx == "pending" {
+            writeln!(
+                f,
+                "Deployed by {}, not yet included in any block",
+                self.data_request_creator_pkh
+            )?;
+        } else {
+            writeln!(
+                f,
+                "Deployed in block {} by {}",
+                self.block_hash_data_request_tx, self.data_request_creator_pkh
+            )?;
+            let data_request_state = self.data_request_state.as_ref().unwrap();
+            let num_commits = self.reveals.as_ref().unwrap().len();
+            let num_reveals = self
+                .reveals
+                .as_ref()
+                .unwrap()
+                .iter()
+                .filter_map(
+                    |(_pkh, reveal, _honest)| {
+                        if reveal.is_empty() {
+                            None
+                        } else {
+                            Some(())
+                        }
+                    },
+                )
+                .count();
+            writeln!(
+                f,
+                "In {} stage, with {} commits and {} reveals",
+                data_request_state.stage, num_commits, num_reveals,
+            )?;
+            writeln!(
+                f,
+                "Commit rounds: {}/{}",
+                data_request_state.current_commit_round,
+                1 + self.data_request_output.extra_commit_rounds
+            )?;
+            writeln!(
+                f,
+                "Reveal rounds: {}/{}",
+                data_request_state.current_reveal_round,
+                1 + self.data_request_output.extra_reveal_rounds
+            )?;
+        }
+
+        if let Some(reveals) = &self.reveals {
+            writeln!(f, "Reveals:")?;
+            for (pkh, reveal, honest) in reveals {
+                let honest_str = if *honest == "rewarded" {
+                    "[Rewarded ]"
```

Comment:
```suggestion
                    "[Rewarded]"
```

Reply:
This is intentional, so that the reveals align:

    [Rewarded ] twit123...
    [Penalized] twit234...

I don't know if this looks better:


    [Rewarded]  twit123...
    [Penalized] twit234...